planning/autoware_behavior_path_start_planner_module/include/autoware_behavior_path_start_planner_module/pull_out_planner_base.hpp

```diff
@@ -40,6 +44,29 @@ enum class PlannerType {
   FREESPACE = 4,
 };
 
+struct PlannerDebugData
+{
+public:
+  PlannerType planner_type;
+  std::vector<std::string> conditions_evaluation;
+  double required_margin{0.0};
+  double backward_distance{0.0};
+  auto str() const
+  {
+    std::stringstream ss;
+    ss << std::left << std::setw(20) << "| Planner type " << std::setw(20) << "| Required margin "
+       << std::setw(20) << "| Backward distance " << std::setw(25) << "| Condition evaluation |"
+       << "\n";
+    for (const auto & result : conditions_evaluation) {
+      ss << std::setw(23) << magic_enum::enum_name(planner_type) << std::setw(23)
+         << (std::to_string(required_margin) + "[m]") << std::setw(23)
+         << (std::to_string(backward_distance) + "[m]") << std::setw(25) << result << "\n";
+    }
+    ss << std::setw(40);
```

Comment:
```suggestion
    ss << std::left << std::setw(20) << "| Planner type" << std::setw(20) << "| Required margin"
       << std::setw(20) << "| Backward distance" << std::setw(25) << "| Condition evaluation |"
       << "\n";
    ss << std::setfill('-') << std::setw(85) << "-" << "\n";
    ss << std::setfill(' ');
    for (const auto & result : conditions_evaluation) {
      ss << "| " << std::setw(18) << magic_enum::enum_name(planner_type)
         << "| " << std::setw(18) << (std::to_string(required_margin) + " [m]")
         << "| " << std::setw(18) << (std::to_string(backward_distance) + " [m]")
         << "| " << std::setw(23) << result << "|\n";
    }
```
By applaying this change, I expect format to be improved a bit... but change is not reflected...
I will investigate tomorrow :sleeping:

Reply:
Let's try those improvements in other PR :bow: